src/renderers/webgl/WebGLUniformsGroups.js

```diff
@@ -0,0 +1,372 @@
+/**
+ * @author Mugen87 / https://github.com/Mugen87
+ */
+
+function WebGLUniformsGroups( gl, info, capabilities ) {
+
+	var buffers = {};
+	var updateList = {};
+
+	var allocatedBindingPoints = [];
+	var maxBindingPoints = ( capabilities.isWebGL2 ) ? gl.getParameter( gl.MAX_UNIFORM_BUFFER_BINDINGS ) : null; // binding points are global whereas block indices are per shader program
+
+	function bind( uniformsGroup, program ) {
+
+		// bind shader specific block index to global block point
+
+		var blockIndex = gl.getUniformBlockIndex( program, uniformsGroup.name );
+		gl.uniformBlockBinding( program, blockIndex, uniformsGroup.__bindingPointIndex );
```

Comment:
Do we need to bind every frame? Or are you gonna optimize later?

Reply:
> Do we need to bind every frame?

As far as I understand it is not necessary if the mapping between binding indices and points did not change. I wanted to enhance this section at a later point, probably via `WebGLState`.

Reply:
Yeah, if I'm right it isn't necessary. Let's optimize later.